confirmation_wizard/wizard/confirmation_wizard.py

```diff
@@ -0,0 +1,122 @@
+# Copyright (C) 2024 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```

Comment:
@ivs-cetmix seems you guys ended up changing the license for LGPL in the manifest file but kept AGPL in the file headers, making the whole inconsistent. Pick the one you want but it should be consistent...

Reply:
> @ivs-cetmix seems you guys ended up changing the license for LGPL in the manifest file but kept AGPL in the file headers, making the whole inconsistent. Pick the one you want but it should be consistent... cc @geomer198

Ooops, just saw this comment. Will fix!